src/win/fs-event.c

```diff
@@ -455,36 +460,34 @@ void uv_process_fs_event_req(uv_loop_t* loop, uv_req_t* req,
             sizew = -1;
           }
 
-          if (filenamew) {
-            /* Convert the filename to utf8. */
+          /* Convert the filename to utf8. */
+          size = WideCharToMultiByte(CP_UTF8,
```

Comment:
IIRC, Bert mentioned he had wanted to phase out `uv__convert_utf16_to_utf8()` in favor of using `WideCharToMultiByte()` directly.